docs/topics/proc_creating-an-application.adoc

```diff
@@ -0,0 +1,38 @@
+:_newdoc-version: 2.15.0
+:_template-generated: 2024-2-21
+:_mod-docs-content-type: PROCEDURE
+
+[id="creating-an-application_{context}"]
+= Creating an application by using MTA
+
+[role="_abstract"]
+You can create a new application by using the {ProductName} {ProductShortName} user interface.
+
+.Prerequisites
+
+* You are logged in to an {ProductShortName} server.
+
+.Procedure
+
+. In the {ProductShortName} user interface, select the *Migration* working mode.
+. Click *Application Inventory* in the left menu bar.
+. Click *Create new*. The *New application* dialog appears.
+. In the form, enter the following information:
+.. *Name*: A unique name for the new application.
+.. *Description*: A short description of the application (optional).
+.. *Business service*: A purpose of the application (optional).
+.. *Tags*: Software tags that characterize the application.
+.. *Owner*: A registered software owner from the drop-down list (optional).
+.. *Contributors*: Contributors from the drop-down list (optional).
+.. *Comments*: Relevant comments on the migrated application.
+. Click *Source code* and enter the following fields:
+.. *Repository type*: *Git* or *Subversion*.
+.. *Source Repository*: A URL of the repository where the software code is saved.
+.. *Branch*: An application code branch in the repository.
+.. *Root path*: [get info] (optional/mandatory).
+. Click *Binary* and enter the following fields:
+.. *Group*: [get info] (optional/mandatory).
```

Comment:
What does [get info] mean here? This looks a bit like a *note to self*. If the problem is that the writer doesn't know what this stands for, I can help here: "The Maven group for the application artifact."

Reply:
Hehe, @rromannissen, no idea, to be honest :D Since I'm only the reviewer, I don't know all the specifics, but it really seems that this is some leftover from the writer's notes :-) Will be fixed.  BTW, what about some definition for Root path, Artifact, and Packaging as well to be consistent in style? Could you please share what these names stand for?

Reply:
also, @rromannissen, what does optional/mandatory means? It's a bit confusing for me as a first-time reader...How can I differentiate between when it's optional and when mandatory?

Reply:
@mpershina good question! The reasoning behind those fields being optional or mandatory is explained [here](https://github.com/konveyor/enhancements/tree/master/enhancements/tackle/tackle-git-maven-integration#description-7). In a nutshell, the Binary section as a whole is optional, but if you enter any value on any of the fields from the section, all of them become mandatory.

Reply:
Alright, @rromannissen, got it! Thanks a lot for the explanation :-) In  this case, what about making the whole "Binary" step optional  with some note in the end? Something like:

```
6. Optional: Click Binary and enter the following fields:

    Group: The Maven group for the application artifact .

    Artifact: The Maven artifact for the application.

    Version: A software version of the application.

    Packaging: The packaging for the application artifact, for example, `JAR`, `WAR`, or `EAR`.

NOTE: If you enter any value in any of the Binary section fields, all fields automatically become mandatory.
```

Reply:
also, @rromannissen , there's another [geti nfo] for the Root path, and again the "optional/mandatory" thing. Could you please provide your explanation here as well? See also the changes I applied to the module from the previous comment. WDYT?